deltacat/utils/pyarrow.py

```diff
@@ -476,10 +476,8 @@ def s3_file_to_parquet(
         content_type != ContentType.PARQUET.value
         or content_encoding != ContentEncoding.IDENTITY
     ):
-        raise ValidationError(
-            f"S3 file with content type: {content_type} and "
-            f"content encoding: {content_encoding} cannot be read"
-            "into pyarrow.parquet.ParquetFile"
+        raise ContentTypeValidationError(
+            content_type=content_type, content_encoding=content_encoding
         )
```

Comment:
Isn't it clearer to make these arguments explicit? We have to look at the message to figure out that we need to pass them.